torch/testing/_creation.py

```diff
@@ -73,20 +70,21 @@ def make_tensor(
             is determined based on the :attr:`dtype` (see the table above). Default: ``None``.
         requires_grad (Optional[bool]): If autograd should record operations on the returned tensor. Default: ``False``.
         noncontiguous (Optional[bool]): If `True`, the returned tensor will be noncontiguous. This argument is
-            ignored if the constructed tensor has fewer than two elements.
+            ignored if the constructed tensor has fewer than two elements. Mutually exclusive with ``memory_format``.
         exclude_zero (Optional[bool]): If ``True`` then zeros are replaced with the dtype's small positive value
             depending on the :attr:`dtype`. For bool and integer types zero is replaced with one. For floating
             point types it is replaced with the dtype's smallest positive normal number (the "tiny" value of the
             :attr:`dtype`'s :func:`~torch.finfo` object), and for complex types it is replaced with a complex number
             whose real and imaginary parts are both the smallest positive normal number representable by the complex
             type. Default ``False``.
-        memory_format (Optional[torch.memory_format]): The memory format of the returned tensor.  Incompatible
-            with :attr:`noncontiguous`.
+        memory_format (Optional[torch.memory_format]): The memory format of the returned tensor. Mutually exclusive
+            with ``noncontiguous``.
 
     Raises:
-        ValueError: if ``requires_grad=True`` is passed for integral `dtype`
-        ValueError: If ``low > high``.
+        ValueError: If ``requires_grad=True`` is passed for integral `dtype`
+        ValueError: If ``low >= high``.
```

Comment:
We allow `low == high`:

https://github.com/pytorch/pytorch/blob/28aa2efd1422e689d3a7dc6b447ca1f31bbad1c3/torch/testing/_creation.py#L120-L121

Reply:
OK but shouldn't this stay `low > high` then? Because this suggests that a `ValueError` will be raised when `low == high`

Reply:
:facepalm: Yes, you are right. Let me revert that.